cypress/integration/AboutMeAndLocation.feature

```diff
@@ -0,0 +1,49 @@
+Feature: About me and and location
+  As a user
+  I would like to add some about me text and a location
+  So others can get some info about me and my location
+
+  The location and about me are show on the user. Later it will be possible
+  to search for users by location.
+
+  Background:
+    Given I am logged in
+
+  Scenario: I change my name
+    When I visit the "settings" page
+    And I enter "Hansi" as my name
+    And I press "Save"
+    And I can click on my profile picture in the top right corner
+    Then I can see a "Hansi" as my name
+
+    When I refresh the page
+    And I can click on my profile picture in the top right corner
+    Then I can see a "Hansi" as my name
+
+    When I visit the "settings" page
+    And I enter "Peter Lustig" as my name
+    And I press "Save"
+
+  Scenario Outline: I set my location to "<location>"
+    When I visit the "settings" page
+    And I enter "<location>" as my location
+    And I press "Save"
+    And I visit the "profile/peter-lustig" page
+    Then I can see a "<location>" as my location
+
+    Examples: Location
+        | location               |
+        | New York               |
+        | Germany                |
+        | Berlin                 |
+        | Mecklenburg-Vorpommern |
```

Comment:
Don't use a scenario outline if you can avoid it. Scenario outlines help you to DRY your code if you have a couple of very similar scenarios. But they are an indicator of too much repetition. Here it is obvious that you don't need to run the same test case 4 times with different inputs. Just don't

Reply:
And you are so sure why? I test it as I had several issues before with different results so I want to test it.

Reply:
I`ve added now one test per type (city, region, country)

Reply:
That's much better. But you should do sth. with the location type. Pimp your cucumber test by adding some assertion. Tell the reader what you expect, what should happen with the type?